### DISCUSSION TITLE
v1.5.0

### DISCUSSION THREAD

**Discussion by Zeal-L** [Zeal-L/BiliBili-Manga-Downloader#99] [Announcements]
![GitHub release (by tag)](https://img.shields.io/github/downloads/Zeal-L/BiliBili-Manga-Downloader/v1.5.0/total)
- **新增功能:**
  - 新增 `Windows` 系统任务栏下载进度显示
  - 新增 `Cbz` 保存格式，该格式与`Zip`类似，但内置`ComicInfo.xml`，是许多漫画站点（如 komga 和 kavita）与浏览软件使用的元数据文件 ([#98][i98])
- **修复bug:**
  - 修复了一个下载完成后可能会导致进度条没有正确被清除的bug
  - 修复了一个导致下载速度错误显示，越来越大的bug
  - 修复了 `BiliPlus` 解析失败未正确返回的bug
  - 修复了软件退出，下载任务仍在后台继续运行的bug
- **优化设置:**
  - 优化需要联网的初始任务处理，大幅度提升程序的启动速度
  - 优化了现有的各种地方的明显界面卡顿，此后尽享丝滑
  - 优化下载速度统计逻辑，现在更加平稳和准确了
  - 本地漫画保存文件夹名称精简（默认不再包含作者名），且现在可以任意命名了 （PS: 元数据文件不能动）
  - 为 `BiliPlus` 下载添加便于识别的请求头，以及其他规范修复
- **开发相关:**
  - 改用 `poetry` 作为包管理器 （原 `pipenv`）
  - 升级 `Python` 到 `3.12` 版本 （原 `3.11`）

[i98]: https://github.com/Zeal-L/BiliBili-Manga-Downloader/pull/98

# 特别感谢
- @Sun-ZhenXing 开源的 [系统任务栏进度条 DLL项目](https://github.com/Sun-ZhenXing/easy-taskbar-progress/tree/main)，为本次更新的新特性提供支持！💖
- @shadlc 尽心竭力的为本项目带来流畅的UI使用体验
- @averageFOSSenjoyer 为漫画下载增添的新保存格式  `Cbz` 

## What's Changed
* ⬆️ Bump rich from 13.5.3 to 13.6.0 by @dependabot in https://github.com/Zeal-L/BiliBili-Manga-Downloader/pull/84
* ⬆️ Bump urllib3 from 2.0.5 to 2.0.6 by @dependabot in https://github.com/Zeal-L/BiliBili-Manga-Downloader/pull/85
* Adding CBZ export option with embedded ComicInfo.xml by @averageFOSSenjoyer in https://github.com/Zeal-L/BiliBili-Manga-Downloader/pull/98
* 修复了一些bug，增加了系统任务栏进度条 by @Zeal-L in https://github.com/Zeal-L/BiliBili-Manga-Downloader/pull/97

## New Contributors
* @averageFOSSenjoyer made their first contribution in https://github.com/Zeal-L/BiliBili-Manga-Downloader/pull/98

**Full Changelog**: https://github.com/Zeal-L/BiliBili-Manga-Downloader/compare/v1.4.0...v1.5.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Zeal-L/BiliBili-Manga-Downloader/releases/tag/v1.5.0'>v1.5.0</a>.</em>

**Comment by saladxuanzhi** [comment]


**Reply by Zeal-L** [reply]
不好意思，是上传时搞反了两个版本的名字，稍等，马上换源

**Reply by Zeal-L** [reply]
@saladxuanzhi 可以重新下载了

**Reply by averageFOSSenjoyer** [reply]
@Zeal-L just fyi the linux version is still stuck on v1.3.2

**Reply by shadlc** [reply]
@averageFOSSenjoyer I apologize for any inconvenience caused. Now it has the correct version. I have re-uploaded and checked it again. Thank you for downloading

**Comment by chinghei1007** [comment]
hmmmm BiliPlus 的 ComicReader 是不是什么东西都没有了，因为解析找不到，网站本身好像也完全没有东西了

**Reply by Zeal-L** [reply]
使用正常啊，网页在这里https://www.biliplus.com/manga/

**Comment by iiihazelssl** [comment]
你好，"BiliPlus访问异常，暂时无法检测是否有效，请自行判断BiliPlus可访问状态或联系开发者"，显示以上这个，要怎么解决？

**Reply by Zeal-L** [reply]
B 站最近修改了缓存图片链接规则，马上会出一个修复版本，请耐心等待

**Reply by iiihazelssl** [reply]
好的，谢谢你
